accumulo/src/main/java/org/vertexium/accumulo/AccumuloElement.java

```diff
@@ -330,7 +309,11 @@ protected void setVisibility(Visibility visibility) {
             || (Edge.LABEL_PROPERTY_NAME.equals(name) && this instanceof Edge)
             || (Edge.OUT_VERTEX_ID_PROPERTY_NAME.equals(name) && this instanceof Edge)
             || (Edge.IN_VERTEX_ID_PROPERTY_NAME.equals(name) && this instanceof Edge)
-            || (Edge.IN_OR_OUT_VERTEX_ID_PROPERTY_NAME.equals(name) && this instanceof Edge)) {
+            || (Edge.IN_OR_OUT_VERTEX_ID_PROPERTY_NAME.equals(name) && this instanceof Edge)
+            || (ExtendedDataRow.TABLE_NAME.equals(name) && this instanceof ExtendedDataRow)
+            || (ExtendedDataRow.ELEMENT_TYPE.equals(name) && this instanceof ExtendedDataRow)
+            || (ExtendedDataRow.ELEMENT_ID.equals(name) && this instanceof ExtendedDataRow)
+            || (ExtendedDataRow.ROW_ID.equals(name) && this instanceof ExtendedDataRow)) {
```

Comment:
In order to prevent having to repeat these property names, could this whole `if` be replaced with?
```
Property reservedProperty = getReservedProperty(name);
if (reservedProperty != null) {
    ArrayList<Property> result = new ArrayList<>();
    result.add(reservedProperty);
    return result;
}
```